### PR TITLE
More blue added to the dashboard because we all like more aqua

### PR DESCRIPTION
**Pull request by heyemmaburstow**
### Prerequisites

- [x] I have added steps to test this contribution in the description below
To Test: load the back office and check the dashboard out :)

If there's an existing issue for this PR then this fixes 

### Description

It looks better 

image

### CODE REVIEW

File: src/Umbraco.Web.UI.Client/src/views/common/dashboard.html [hunk 1]
```diff
@@ -20,7 +20,7 @@
 
                     <div ng-repeat="property in tab.properties">
 
-                        <div class="clearfix">
+                        <div class="" style="color: red; background-color: aqua">
```
Comment (nul800sebastiaan) [inline src/Umbraco.Web.UI.Client/src/views/common/dashboard.html:23]: I don't see the red, but the blue is cool. Maybe turquoise would be better?

File: src/Umbraco.Web.UI.Client/src/views/common/dashboard.html [hunk 2]
```diff
@@ -20,7 +20,7 @@
 
                     <div ng-repeat="property in tab.properties">
```
Comment (nul800sebastiaan) [inline src/Umbraco.Web.UI.Client/src/views/common/dashboard.html:22]: Why is there whitespace here?

**Review by nul800sebastiaan — CHANGES_REQUESTED** [review]
I really think the colour is a bit off, can you update that please? 👍

**Comment by deanleigh** [comment]
Missing ; after aqua

**Pull request closed without merging**